src/pages/individual/Individual.jsx

```diff
@@ -117,6 +123,38 @@ export default function Individual() {
     },
     [individualData, fieldSchemas],
   );
+  const encounters = get(individualData, 'encounters', []);
+
+  const assetSources = useMemo(
+    () => {
+      const combinedAssets = reduce(
+        encounters,
+        (memo, encounter) => {
+          const annotations = get(encounter, 'annotations');
```

Comment:
falling back to `[]` here would prevent you from hitting an error on the next line if there are no annotations for some reason

Reply:
Done